report_pre_thesis/src/sections/experiments/other_models.tex

```diff
@@ -0,0 +1,56 @@
+\subsection{Experiment: Other Models}\label{sec:experiment_other_models}
+\citet{cleggRecalibrationMarsScience2017} have only compared their new approach with the original method presented by \citet{wiensPreFlight3}, and have not conducted experiments using alternative methods to establish the superiority of their chosen approach.
+Therefore, we decided to compare the performance of the PLS1-SM and ICA models to other models.
+The objective is to evaluate two distinct scenarios.
+In the first scenario, we aim to conduct a direct comparison between the MOC model and an alternative model. The second scenario revolves around substituting either PLS or ICA with a different model and then calculating a weighted average.
+We have decided to conduct the experiments using the following models:
+
+\begin{itemize}
+	\item \textbf{XGBoost}, a gradient boosting algorithm \cite{chen_xgboost_2016}
+	\item \textbf{ANN}, a neural network model
+\end{itemize}
+
+Building on the findings of \citet{andersonPostlandingMajorElement2022}, who demonstrated Gradient Boosting Regression (GBR) models' superior predictive accuracy in analyzing major oxide compositions in geological samples, we adopt XGBoost for its refined GBR implementation.
+This decision is informed by XGBoost's documented success in enhancing model precision, as detailed in Section~\ref{sec:related_works}.
+
+In light of \citeauthor{takahashi_quantitative_2017}'s findings on the limitations of traditional LIBS calibration methods and the potential of ANNs to address non-linearities in solid sample analysis, described in Section~\ref{sec:related_works}, we also decided to include ANNs in our experiments.
+This approach is motivated by ANNs' capacity to discern complex, non-linear relationships within LIBS spectra.
+
+The ANN comprises of a sequence of fully connected (dense) layers and dropout layers, aimed at reducing overfitting through regularization.
+The architecture begins with an input layer of 6144 units, reflecting the high-dimensional nature of the input data.
```

Comment:
Its true that it reflects the high-dimensional nature of the input layer, but perhaps we should explain that we chose 6144 because that is equal to the number of wavelengths in our data? Otherwise it just seems like some arbitrary high number.

Reply:
Done